test/audits/data-table-headers-missing-test.js

```diff
@@ -0,0 +1,285 @@
+module('DataTableHeadersMissing');
+
+function buildRow(cells) {
```

Comment:
This is slick, but could you:
1) Scope using an IIFE
1) JSDoc these methods (like https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/test/audits/multiple-aria-owners-test.js#L18)